server/.env

```diff
@@ -0,0 +1,3 @@
+TEAM_MEMBERS="Philippe, Alan, Bonnie, Shums Kassam"
+MONGO_URI = mongodb+srv://guohao123:guohao123@guohao.wbjfm.mongodb.net/team-dolphin?retryWrites=true&w=majority&ssl=true
```

Comment:
@Jovialiste82 You'll need to update this and be careful not to push it to Github

Reply:
I have to admit I am still a bit confused about this one. I ran into some issues with this .env file yesterday after pulling dev into my machine and then merging dev into my branch. Jess helped me run a few git commands about this (see history enclosed) And right now I am positive that i have server/.env on my .gitignore file but i just pushed new commits a few minutes ago (to run prettier and  remove some test code) and .env still shows up in my branch. Really no idea how to fix this...
[TerminalHistory.txt](https://github.com/hatchways/team-dolphin/files/6130601/TerminalHistory.txt)

Reply:
Handled with Jess,  I think it's removed now